src/QsCompiler/TestProjects/test1/test1.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
I thought the goal of some of these test projects was to ensure we didn't break existing projects using older targets. Is that not the case?

Reply:
`netcoreapp2.1` is no longer supported by .NET so it stands to reason we should be doing extra work to support it either. This was also part of the PR https://github.com/microsoft/qsharp-compiler/pull/1455.